src/reference/01-General-Info/04-Bintray-For-Plugins.md

```diff
@@ -56,45 +56,41 @@ publish to bintray.
 
 First, add the bintray-sbt to your plugin build.
 
-First, create a `project/bintray.sbt` file
+To do that, create a `project/bintray.sbt` file
 
 ```scala
 addSbtPlugin("me.lessis" % "bintray-sbt" % "$bintray_sbt_version$")
 ```
 
-Next, a make sure your `build.sbt` file has the following settings
+Next, ensure your `build.sbt` file has the following settings
 
 ```scala
-import bintray.Keys._
-
-lazy val commonSettings = Seq(
-  version in ThisBuild := "<YOUR PLUGIN VERSION HERE>",
-  organization in ThisBuild := "<INSERT YOUR ORG HERE>"
-)
-
-lazy val root = (project in file(".")).
-  settings(commonSettings ++ bintrayPublishSettings: _*).
-  settings(
-    sbtPlugin := true,
-    name := "<YOUR PLUGIN HERE>",
-    description := "<YOUR DESCRIPTION HERE>",
-    // This is an example.  bintray-sbt requires licenses to be specified 
-    // (using a canonical name).
-    licenses += ("Apache-2.0", url("https://www.apache.org/licenses/LICENSE-2.0.html")),
-    publishMavenStyle := false,
-    repository in bintray := "sbt-plugins",
-    bintrayOrganization in bintray := None
-  )
+version := "0.1.0" // change to suit
```

Comment:
Requiring a multi-project build means more complexity than is required for most users and is offputting. Best not to conflate the concepts of "how to package and use an SBT plugin" and "how multi-project builds work". 

I think that an example of a single-project build and another example of a multi-project build would be ideal. This would mean two subsections in the docs, with a common preamble section. That way the general concepts can be explained, followed by two discussions of how the concepts can be applied.